src/root/array.d

```diff
@@ -192,6 +192,22 @@ public:
         assert(a <= b && b <= dim);
         return data[a .. b];
     }
+
+    static if (is(T == const(char)*))
```

Comment:
Array does not provide canFind for other types.
Adding it only for char* is very confusing.